test/fixtures/test-runner/output/eval_dot.snapshot

```diff
@@ -1 +1,11 @@
 .X
+✖  Failed tests:
```

Comment:
I think there might be a space too much before `Failed tests:`

Also, I'm would remove the x before and add a newline after the dots.

Reply:
I have added a newline after the colon. Did you mean a new empty line?
```suggestion
Failed tests:
x name_of_the_test
```
or
```suggestion
Failed tests:

x name_of_the_test
```